src/components/utils/TestMergeRow.tsx

```diff
@@ -194,7 +194,7 @@ export default function TestMergeRow({
                                                 e.shiftKey
                                                     ? onSelectCommit(
                                                           pr.head,
-                                                          "No comment set - Fast Update"
+                                                          testmergeinfo?.comment ?? null
```

Comment:
What's the point of this?

Reply:
This keep the existing comment if there is one

Reply:
No, what's the point of the ?? null

Reply:
`comment?: string | null` into a field that is `string | null`